schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,707 @@
+          },
+          "is_submitter": {
+            "description": "Indicates whether this person is the submitter of this submission",
+            "type": "boolean"
+          }
+        },
+        "required": [
+          "full_name",
+          "last_name"
+        ],
+        "$comment": "Removed requirement for first name. Some people do not have a last name--optional too?"
+      },
+      "arxiv_status": {
+        "title": "arXiv posting status",
+        "description": "The status of this document version on arXiv.",
+        "type": "string",
+        "enum": ["posted", "withdrawn"],
+        "minLength": 1
+      },
+      "external_publication_status": {
+        "title": "External publication status",
+        "description": "The publication status of this document version.",
+        "type": "string",
+        "enum": ["preprint", "submitted", "accepted", "published"],
+        "$comment": "Is this enough?"
+      },
+      "reference_string": {
+        "title": "Reference string",
+        "description": "An unstructured reference string.",
+        "type": "string",
+        "minLength": 1
+      },
+      "source_info": {
+        "title": "Source information",
+        "description": "Information about the source input.",
+        "type": "object",
+        "properties": {
+          "source_format": {
+            "title": "Source format",
+            "type": "string",
+            "enum": ["tex", "pdftex", "pdf", "ps", "html", "docx", "none"]
+          },
+          "legacy_source_flags": {
+            "title": "Legacy source flags",
+            "type": "string",
+            "enum": ["1", "1A", "1S", "A", "AD", "AS", "D", "S", "S1"]
+          },
+          "size_kilobytes": {
+            "title": "Source size in kilobytes",
+            "description": "The size of the source file, in kilobytes.",
+            "type": "number"
+          },
+          "checksum": {
+            "title": "Checksum",
+            "description": "Checksum of the source file, if source file exists.",
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "algorithm": {
+                  "type": "string",
+                  "enum": ["MD5", "SHA-1", "SHA-256", "SHA-512"]
+                },
+                "value": {
+                  "type": "string",
+                  "pattern": "^[0-9a-f]+$"
+                }
+              },
+              "required": [
+                "algorithm",
+                "value"
+              ],
+              "additionalProperties": false
+            },
+            "minItems": 1
+          }
+        },
+        "required": [
+          "source_format",
+          "size_kilobytes"
+        ],
+        "additionalProperties": false
+      },
+      "titles": {
+        "title": "List of titles",
+        "description": "The English and non-English versions of the title. The English title must be provided first.",
```

Comment:
What about the format of the title? the current situation where it might or might not contain LaTeX is not really satisfactory. Maybe we should discuss it separately at a later stage.

Reply:
One way we've handled that elsewhere (document mapping in search) is to have a `title_utf8` field where the texisms have been converted to their utf-8 equivalents. We could do something similar in this schema.